Lesson1HomeTask/task1/main.cpp

```diff
@@ -0,0 +1,112 @@
+#include <iostream>
+#include <vector>
+#include <string>
+#include <algorithm>
+
+using my_vec_t = std::vector<int>;
+
+my_vec_t my_merge1 (const my_vec_t& v1, const my_vec_t& v2)
+{
+    my_vec_t result {v1};
+    const auto  result_size = v1.size() + v2.size();
+    result.resize(result_size);
+
+    auto it = std::next (result.begin(), v1.size());
+
+    std::copy (v2.begin(), v2.end(), it);
+    return result;
+}
+
+my_vec_t my_merge2 (const my_vec_t& v1, const my_vec_t& v2)
+{
+    my_vec_t result {v1};
+    for (const auto& el : v2)
```

Comment:
Also please use more understandable names for variables, functions, etc.

Reply:
ok